background.ts

```diff
@@ -43,10 +43,14 @@ import * as window from '@pkg/window';
 import { closeDashboard, openDashboard } from '@pkg/window/dashboard';
 import { preferencesSetDirtyFlag } from '@pkg/window/preferences';
 
-Electron.app.setName('Rancher Desktop');
 Electron.app.setPath('cache', paths.cache);
 Electron.app.setAppLogsPath(paths.logs);
 
+if (!Electron.app.requestSingleInstanceLock()) {
```

Comment:
I would move both this line and the call to `clearLoggingDirectory()` to the line after `console` is defined, so that way if something is logged it goes to `background.log` and not the console (which for a UI-based launch is hard to find, if not non-existent).

Reply:
Makes sense. Making this change.